pkg/api/types.go

```diff
@@ -1739,6 +1744,9 @@ type ReplicationControllerStatus struct {
 	// The number of ready replicas for this replication controller.
 	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
 
+	// The number of available replicas for this replication controller.
```

Comment:
explain what "available" means in the comment (how is it different from ready ones, and mention minreadyseconds)